### ISSUE TITLE
State variable in UILabel body leads to poor performance

### ISSUE THREAD

**Issue by rcstuber** [schibsted/layout#122]
I experienced when using a state variable inside a UILabel's body text instead of using the `title` property execution gets extremely slow, especially when using inside `UITableViewCell` with many rows.

```xml
<UITableViewCell height="60" reuseIdentifier="Cell">
    <UILabel>{name}</UILabel> // <--- extremely slow, hangs UI thread
    <UILabel text="{name}"/>  // <--- fast
</UITableViewCell>
```
Why would I use the body instead of text property? I.e. to make use of the attributed text markup features `<UILabel><b>{name}</b></UILabel>`

I assume it has to do with the (X)HTML parser, though I did not trace it.
Thanks

**Comment by rcstuber** [comment]
I now measured execution time  of the `node.setState` in a `tableView:cellForRow` (averaged over many calls while scrolling) once using the `text` property and once the rich text support. It proved my point.

Rich Text: ~ 0.013 ms / per call
Text Property: ~0.0033 ms / per call

So there is a magnitude of ~4x here on the iOS Simulator. I did that in the Sample App so it's not somewhere in my code, simply by adding a UILabel to the inline cell in the `Table.xml` and passing a string in.

Cheers
